integration_tests/rw-kafka-sink-db/docker-compose-local.yml

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
what is the difference between local and the other version

Reply:
The local version requires a local running RisingWave and is for test for a newly built RisingWave that is not in official docker image yet.

Reply:
then please remove this part. it does not always work on local machine, for example, macos, local network and docker network are isolated by default.